test/VisionModel/resnet.jl

```diff
@@ -1,12 +1,11 @@
+using Flux
+using FluxModels
+using Test
+
 @testset "ResNet" begin
-  for MODEL in [
-    ResNet18,
-    ResNet34,
-    ResNet50,
-    ResNet101,
-    ResNet152,
-    ]   
-    model = MODEL()
-    @test size(model(rand(256, 256, 3, 50))) == (1000, 50)
-  end
+    @test size(ResNet18(rand(256, 256, 3, 50))) == (1000, 50),
+    @test size(ResNet34(rand(256, 256, 3, 50))) == (1000, 50),
+    @test size(ResNet50(rand(256, 256, 3, 50))) == (1000, 50),
+    @test size(ResNet101(rand(256, 256, 3, 50))) == (1000, 50),
+    @test size(ResNet152(rand(256, 256, 3, 50))) == (1000, 50)
```

Comment:
You first need to instantiate the model, then call it. You are trying to pass the input directly into the constructor right now. You could do something like:
```julia
@testset "ResNet" for model in [ResNet18, ResNet34, ResNet50, ResNet101, ResNet152]
    m = model()
    @test size(m(rand(256, 256, 3, 50))) == (1000, 50)
end

Reply:
I had done the same, it was showing ResNet18 not defined errors.

Reply:
```
Test threw exception
  Expression: size(m(rand(256, 256, 3, 50))) == (1000, 50)
  MethodError: objects of type Array{Any,1} are not callable
  Use square brackets [] for indexing an Array.
```
this is error.

Reply:
I think the problem is something like inplanes, outplanes not defined properly